int8[] dist

float32 deg
float32 dist_y